### PR TITLE
add a context menu for running sky app

### PR DESCRIPTION
**Pull request by keertip**
@devoncarew 

start of a framework for launching and managing sky apps.

### CODE REVIEW

File: lib/sky/run_app.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+class RunSkyAppContextCommand extends ContextMenuItem {
+  RunSkyAppContextCommand(String label, String command) : super(label, command);
+
+  bool shouldDisplay(AtomEvent event) {
+    _logger.info('showing sky menu');
+    String filePath = event.targetFilePath;
+    _logger.info('filepath = $filePath');
+    DartProject project = projectManager.getProjectFor(filePath);
```
Comment (devoncarew) [inline lib/sky/run_app.dart]: we should check for a null project here

File: lib/sky/run_app.dart [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      return result.stdout;
+    });
+  }
+}
+
+class RunSkyAppContextCommand extends ContextMenuItem {
+  RunSkyAppContextCommand(String label, String command) : super(label, command);
+
+  bool shouldDisplay(AtomEvent event) {
+    _logger.info('showing sky menu');
```
Comment (devoncarew) [inline lib/sky/run_app.dart]: we probably don't need logging here, when checking to see if a context menu item should be contributed.

**Comment by devoncarew** [comment]
lgtm! two suggestions

File: lib/sky/run_app.dart (hunk 2 above)
Comment (keertip) [inline lib/sky/run_app.dart]: Missed that! removed.

File: lib/sky/run_app.dart (hunk 1 above)
Comment (keertip) [inline lib/sky/run_app.dart]: done

**Pull request merged**